imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -62,33 +131,24 @@ def test_lo_ialirt():
         / "imap_codice_lo-ialirt_20250814_v001.pkts"
     )
 
-    # # Validation
-    # val_path = (
-    #     imap_module_directory
-    #     / "tests/codice/data/l1a_validation/"
-    #     / "imap_codice_l1a_lo-ialirt_20250807174600_v0.0.3.cdf"
-    # )
-    # val_data = load_cdf(val_path)
-    # print(val_data)
+    # Validation
+    val_path = (
+        imap_module_directory
+        / "tests/codice/data/l1a_validation/"
+        / "imap_codice_l1a_lo-ialirt_20250814211100_v0.0.5.cdf"
+    )
+    val_data = load_cdf(val_path)
 
     processed_data = process_codice_l1a(file_path=test_file_path)[0]
-    for variable in processed_data:
-        if variable in [
-            "rgfo_half_spin",
-            "nso_half_spin",
-            "sw_bias_gain_mode",
-            "st_bias_gain_mode",
-            "data_quality",
-            "spin_period",
-        ]:
-            assert processed_data[variable].shape == (8,)
-        # For energy dimensions
-        elif variable in ["energy_table", "acquisition_time_per_step"]:
-            assert processed_data[variable].shape == (128,)
-        elif variable == "k_factor":
-            assert processed_data[variable].shape == (1,)
-        else:
-            assert processed_data[variable].shape == (8, 128, 1)
+    for variable in val_data.data_vars:
+        if variable in EXPECTED_MISMATCHES or variable.startswith(
+            UNCERTAINTY_VARIABLES
+        ):
+            continue
+        assert processed_data[variable].shape == val_data[variable].shape, (
+            f"Shape mismatch for variable '{variable}'"
```

Comment:
Just to be clear, this is still only validating the shapes and not the values. Is that what you want for now because the values still don't match?